gen/src/MPIgenerator.jl

```diff
@@ -1,4 +1,5 @@
 module MPIgenerator
+    __precompile__(false)
```

Comment:
We only need to update `Clang.jl`, with no other changes to the script?

Reply:
Yeah, I can remove this now, since this was used at early stage when monkey patching.